core/src/main/java/com/linecorp/armeria/common/AbstractRequestContextBuilder.java

```diff
@@ -78,6 +78,8 @@
     private MeterRegistry meterRegistry = NoopMeterRegistry.get();
     @Nullable
     private EventLoop eventLoop;
+    @Nullable
+    private EventLoop serviceWorkerGroup;
```

Comment:
Do we need `serviceWorkerGroup` for `RequestContext`?
In `RequestContext` point of view, a proper `EventLoop` either 
a network IO loop or a service loop should be chosen by the caller with the context being built.